CHANGELOG.md

```diff
@@ -291,4 +291,16 @@ All notable changes to this project will be documented in this file. The format
 ### Fixed
  - `hostrule` with `sslKeyCertificate` of type `secret` can now be configured in all namespaces in OpenShift clusters.
  - Fixed an issue of VS creation failing with 470 Ingress with a single path giving a 470 error.
- - `nodeNetworkList` is not mandatory for NSX-T Overlay deployments.
\ No newline at end of file
+ - `nodeNetworkList` is not mandatory for NSX-T Overlay deployments.
+
+ ## AKO-1.8.2
+
+### Changed
+- Logs, from Istio file watcher, are changed from `Error` level to `Warn` level.
```

Comment:
Do we have to mention this in the changelog? or do we have to elaborate on why we have made this change?

Reply:
As per Arif, it will be required to include as it is reported from field and it will be good to notify that required change has been made.